src/platform/plugins/shared/discover/public/embeddable/initialize_search_embeddable_api.tsx

```diff
@@ -197,38 +199,30 @@ export const initializeSearchEmbeddableApi = async (
       canEditUnifiedSearch,
     },
     stateManager,
+    anyStateChange$: onAnyStateChange.pipe(map(() => undefined)),
```

Comment:
Curious how this works, what does mapping to `undefined` do here?

Reply:
`anyStateChange$` is typed as `Observable<void>`. `onAnyStateChange` is an`Observable<SearchEmbeddableSerializedAttributes>`. Using `pipe` and `map` convert emit from `SearchEmbeddableSerializedAttributes` to `void`.